src/flashcart/ed64/ed64.c

```diff
@@ -15,45 +15,79 @@
 
 // #include "../menu/settings.h"
 
-// // This is a trial hack before using the settings API.
-// #ifndef LAST_ROM_FILE_PATH
-// #define LAST_ROM_FILE_PATH "/ed64/last_rom.txt"
-// #endif
-
+// This is a trial hack before using the settings API.
+#ifndef LAST_SAVE_FILE_PATH
+#define LAST_SAVE_FILE_PATH strip_sd_prefix("/menu/last_rom.txt")
+#endif
 
 extern int ed_exit(void);
 
 static flashcart_err_t ed64_init (void) {
 
-    // TODO: partly already done, see https://github.com/DragonMinded/libdragon/blob/4ec469d26b6dc4e308caf3d5b86c2b340b708bbd/src/libcart/cart.c#L1064
-
-    // FIXME: Update firmware if needed.
-    // FIXME: Enable RTC if available.
+    if (file_exists(strip_sd_prefix("/menu/RESET"))) { // checks if reset
+    
+    f_unlink(strip_sd_prefix("/menu/RESET"));
```

Comment:
To make this easier to maintain and change, use a define (as done with LAST_SAVE_FILE_PATH)